jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -358,6 +468,9 @@ YUI.add('user-profile-entity', function() {
         });
         destroyButton = this._generateDestroyButton(entity);
       } else {
+        if (isCharm) {
```

Comment:
Everything else about this component seems to occur here; I'm not sure it will be clear in componentWillMount, nor does it seem worth it to add that for this.

Thoughts?